src/main/java/seedu/address/logic/commands/FlipCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Flips the current flashcard under review to show its answer to the user.
+ */
+public class FlipCommand extends Command {
+
+    public static final String COMMAND_WORD = "flip";
```

Comment:
Small comment but can we do `on enter` as flip or type `f` instead of `flip`?  Cause may be a bit slow for the user to keep typing `flip` + enter

Reply:
Yep, agreed. We can implement `on enter` in future.